core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -161,11 +163,25 @@ class TechManager {
                 getRuleset().technologies[currentTechnologyName()]!!.cost))
     }
 
+    private fun scienceFromResearchAgreements(): Int {
```

Comment:
having a parameter and a function with the exact same name can lead to confusion, please rename one of them